### PR TITLE
Document `subctl cloud prepare azure` command `air-gapped` flag

### PR DESCRIPTION
**Pull request by tpantelis**

### CODE REVIEW

File: src/content/community/releases/_index.en.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ weight = 40
 
 ## v0.16.0
 
+* The `subctl cloud prepare azure` command now has a new flag, `air-gapped`, to indicate the cluster is in an air-gapped
```
Comment (skitt) [inline src/content/community/releases/_index.en.md]:
```suggestion
* The `subctl cloud prepare azure` command has a new flag, `air-gapped`, to indicate the cluster is in an air-gapped
```

“now“ and “new” seem redundant here.

**Review by skitt — CHANGES_REQUESTED** [review]
This targets the wrong branch, `release-notes_0.16` has been merged. I’ll fix the PR.

File: .github/workflows/branch.yml [hunk 2]
```diff
@@ -11,6 +11,6 @@ jobs:
     name: PR targets branch
     runs-on: ubuntu-latest
     steps:
-      - name: Check that the PR targets devel
-        if: ${{ github.base_ref != 'devel' }}
+      - name: Check that the PR targets devel or a release note branch
```
Comment (skitt) [inline .github/workflows/branch.yml]: Let’s do #954 instead.

**Review by skitt — APPROVED** [review]

**Review by skitt — APPROVED** [review]

**Review by nyechiel — APPROVED** [review]

**Comment by dfarrell07** [comment]
FYI, I can't merge these PRs into non-devel branches.

**Pull request merged**